chapter_preliminaries/ndarray_vn.md

```diff
@@ -226,7 +226,12 @@ These apply a standard scalar operation to each element of an array.
 For functions that take two arrays as inputs, elementwise operations apply some standard binary operator on each pair of corresponding elements from the two arrays. We can create an elementwise function from any function that maps from a scalar to a scalar.
 -->
 
-*dịch đoạn phía trên*
+Cuốn sách này không nói về kỹ thuật phần mềm.
+Chúng tôi không chỉ hứng thú với việc đơn giản đọc và ghi dữ liệu vào/từ các mảng mà còn muốn thực hiện các phép toán toán học trên 
+các mảng này.
+Một vài phép toán đơn giản và hữu ích nhất là các phép toán tác động lên *từng thành phần* (*elementwise*).
```

Comment:
```suggestion
Một vài phép toán đơn giản và hữu ích nhất là các phép toán tác động lên *từng phần tử* (*elementwise*).
```

@lkhphuc từ này được thống nhất là *từng phần tử* sau khi anh dịch phần này. Anh sẽ sửa lại toàn bộ.